Geometry/EcalCommonData/plugins/dd4hep/DDEcalBarrelNewAlgo.cc

```diff
@@ -0,0 +1,2179 @@
+#include "DD4hep/DetFactoryHelper.h"
+#include "DD4hep/Printout.h"
+// FIXME: use local definition until dd4hep fixes the uniits
+//#include "DataFormats/Math/interface/GeantUnits.h"
+#include "DetectorDescription/DDCMS/interface/DDPlugins.h"
+#include "DetectorDescription/DDCMS/interface/BenchmarkGrd.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "Geometry/CaloGeometry/interface/EcalTrapezoidParameters.h"
+#include "Math/AxisAngle.h"
+#include "CLHEP/Geometry/Point3D.h"
+#include "CLHEP/Geometry/Vector3D.h"
+#include "CLHEP/Geometry/Transform3D.h"
+
+using namespace std;
+using namespace cms;
+using namespace dd4hep;
+
+// FIXME: use local definition until dd4hep fixes the uniits
+// using namespace geant_units::operators;
+
+constexpr long double piRadians(M_PIl);              // M_PIl is long double version of pi
+constexpr long double degPerRad = 180. / piRadians;  // Degrees per radian
+constexpr double operator"" _deg(long double deg) { return deg / degPerRad; }
+constexpr double operator"" _deg(unsigned long long int deg) { return deg / degPerRad; }
+
+// Length
+constexpr double operator"" _mm(long double length) { return length * 0.1; }
```

Comment:
here ```_mm ``` means scaling by 1/10, while in DataFormats/Math will mean multiplying by one